model/src/channel/embed/author.rs

```diff
@@ -18,7 +22,7 @@ mod tests {
         let value = EmbedAuthor {
             icon_url: Some("https://example.com/1.png".to_owned()),
             name: Some("test".to_owned()),
-            proxy_icon_url: None,
+            proxy_icon_url: Some("https://example.com".to_owned()),
```

Comment:
Let's not modify existing test cases and only modify the output. This should be a new test